javagen/src/main/java/com/azure/autorest/util/returntype/ReturnTypeDescriptionAssembler.java

```diff
@@ -100,7 +100,7 @@ public boolean accept(IType returnType) {
         public String handle(String description, GenericType returnType, IType baseType) {
             String assembledDesc;
             if (description == null) {
-                if (PrimitiveType.Void == baseType) { // Response<Void>
+                if (PrimitiveType.Void == baseType || PrimitiveType.Void == returnType.getTypeArguments()[0]) { // Response<Void>
```

Comment:
The reason adding `PrimitiveType.Void == returnType.getTypeArguments()[0]` condition? And why it is not `ClassType.Void`?

Reply:
In situations like `create simple async rest response`, `syncReturnType` is `PrimitiveType.Void` and is passed in as `baseType` parameter. `ClassType.Void` is more semantically suitable for `baseType` here and I added `baseType.asNullable()` to handle this situation.